src/SmartComponents/SystemAdvisor/SystemAdvisor.cy.js

```diff
@@ -162,4 +163,73 @@ describe('system rules table', () => {
       cy.get('.ins-c-primary-toolbar__first-action').should('not.exist');
     });
   });
+  
+  describe('BulkSelector', () => {
+    function selectRandomEnabledRows({
+      rows,
+      numberOfRowsToSelect,
+    }) {
+      const enabledRows = Array.from(rows).filter(row => {
+        const checkbox = row.querySelector('input[type="checkbox"]');
+        if(!checkbox.hasAttribute('disabled')){
+          return true;
+        }
+      })
+      const rowCount = enabledRows.length
+
+      const randomIndices = [];
+      while (randomIndices.length < numberOfRowsToSelect) {
+        const randomIndex = Math.floor(Math.random() * rowCount);
+        if (!randomIndices.includes(randomIndex)) {
+          randomIndices.push(randomIndex);
+        }
+      }
+
+      randomIndices.forEach(index => {
+        enabledRows[index].querySelector('input[type="checkbox"]').click();
+      });
+    }
+
+    it(`The Bulk selector shows the correct number of systems selected.`, () => {
+      
+      // check that empty
+      cy.get(':nth-child(2) > .pf-v5-c-menu-toggle').should('have.text', '')
+      
+      // select a couple
+      //  but only ones that can be selected
+      cy.get('.pf-v5-c-table__tbody').then(rows => {selectRandomEnabledRows({rows: rows, numberOfRowsToSelect: 3})});
+      
+      // check that it shows correct number
+      cy.get(':nth-child(2) > .pf-v5-c-menu-toggle').should('have.text', '3 selected')
+
+      // Select None
+      cy.get(':nth-child(2) > .pf-v5-c-menu-toggle > .pf-v5-c-menu-toggle__controls').click();
```

Comment:
PT_BULK_SELECT results in a flaky test. I think it's because it targets the `pf-v5-c-menu-toggle` element, instead of the `pf-v5-c-menu-toggle__controls` element. In some cases it then doesn't open the BS menu